### ISSUE TITLE
Documentation on i18n

### ISSUE THREAD

**Issue by jwayong** [ING-Group/tpa-bootstrap#14]
- How to do i18n
- i18n code structure dynamic and design time

**Comment by josh-bradley** [comment]
Will add a demo and readme for the dynamic stuff. I'm not sure what the strategy is for design time

**Comment by abowen** [comment]
Is there a need for design time? Why not just only include 1 dictionary and hide the language switcher?

**Comment by josh-bradley** [comment]
Makes sense to me I just thought @barthinho had done something last week towards setting up design time switching.
